pages/docs/index.html

```diff
@@ -3,5 +3,144 @@
+
+<pre>
+  Fedora 27 (Workstation Edition) 27.72 (ostree)
+</pre>
+
+  Currently, layered packages are not easily identified identified in the grub
+  menu. A fix for this is being discussed.</p>
+
+  <h3>Update my system?</h3>
+
+  <p>To get updates for the OS, you can run: <em>rpm-ostree upgrade</em>.
+  Starting with Fedora 28, gnome-software will also offer OS
+  updates on Atomic systems. Note that due to the atomic nature
+  of the OS, you will have to reboot into the new image to
+  have the updates take effect.</p>
+
+  <h3>Check for system updates?</h3>
+
+  <p>This command will let you check if updates are available: <em>rpm-ostree upgrade --check</em>.
+  Starting with Fedora 28, gnome-software will also notify you about OS updates, just
+  like it does on the normal workstation.</p>
+
+  <h3>Update my applications?</h3>
+
+  <p>To update your flatpak applications, you can run: <em>flatpak update</em>.
+  gnome-software will also offer flatpak application updates.
+  Note that flatpaks are independent of the base system, so you
+  can install updates without a reboot, and just restart affected
+  applications afterwards to get the new version.</p>
+
+  <h3>Check for application updates?</h3>
+
+  <p>This command will let you check if updates are available: <em>flatpak remote-ls --updates</em>.
+  if you add the -d option, it will generate detailed information about the available
+  updates:
+
+<pre>
+$ flatpak remote-ls --updates -d
+Ref                                                     Origin             Commit       Installed size Download size
+app/com.valvesoftware.Steam/x86_64/stable               flathub            effe21b05e42   6.3 MB         4.1 MB
+app/org.blender.Blender/x86_64/stable                   flathub            ac90294a1cc3 225.4 MB        79.6 MB
+runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6 flathub            818a1504fad1   8.7 MB         2.7 MB
+runtime/org.gnome.Platform/x86_64/3.24                  flathub            6132154b089b 739.5 MB       281.4 MB
+runtime/org.gnome.Platform/x86_64/3.26                  flathub            be9fe7b43711 782.5 MB       296.8 MB
+runtime/org.gnome.Sdk/x86_64/3.26                       flathub            c62d5ddf669a   1.6 GB       582.9 MB
+runtime/org.kde.Platform/x86_64/5.9                     flathub            ea891f4a918f   1.1 GB       391.0 MB
+app/org.gnome.Builder/x86_64/master                     gnome-apps-nightly b14e6c0fc6c3 105.0 MB        35.4 MB
+</pre>
+
+  gnome-software will also notify you about application updates, just like it does
+  on the normal workstation.</p>
+
+  <h3>Get onto a new OS release? Do I have to reinstall?</h3>
+
+  <p>No, you don't have to reinstall. You can run: <em>rpm-ostree rebase</em>,
+  and give it the name of the new OS (fixme: how to find that name).
+  The process is very similar to a system update: the new OS is
+  downloaded and installed in the background, and you just boot into
+  it when it is ready.</p>
```

Comment:
Would it be worth to point that this is a reversible action, and that if the upgrade is not ok, people can just come back to the previous OS (since that's a big feature)